### PR TITLE
Its a ship, not a station! Ship goal edition

### PR DESCRIPTION
**Pull request by SharkeeByond**
## About The Pull Request

Changes some of the text in station goal announcements so it suits NSV13

## Why It's Good For The Game

Muh immershuns

## Changelog
tweak: Tweaked station goals so they say ''ship'' instead of ''station'' now.
/:cl:

### CODE REVIEW

File: code/modules/station_goals/bsa.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 /datum/station_goal/bluespace_cannon/get_report()
 	return {"Our military presence is inadequate in your sector.
-	 We need you to construct BSA-[rand(1,99)] Artillery position aboard your station.
+	 We need you to construct BSA-[rand(1,99)] Artillery position aboard your ship.
```
Comment (Karmic-Skink) [inline code/modules/station_goals/bsa.dm:10]: //NSV13 tag required

File: code/modules/station_goals/dna_vault.dm [hunk 2]
```diff
@@ -33,7 +33,7 @@
 
 /datum/station_goal/dna_vault/get_report()
 	return {"Our long term prediction systems indicate a 99% chance of system-wide cataclysm in the near future.
-	 We need you to construct a DNA Vault aboard your station.
+	 We need you to construct a DNA Vault aboard your ship.
```
Comment (Karmic-Skink) [inline code/modules/station_goals/dna_vault.dm:36]: And here.

File: code/modules/station_goals/shield.dm [hunk 3]
```diff
@@ -6,7 +6,7 @@
 	var/coverage_goal = 500
 
 /datum/station_goal/station_shield/get_report()
-	return {"The station is located in a zone full of space debris.
+	return {"The ship is located in a zone full of space debris.
```
Comment (Karmic-Skink) [inline code/modules/station_goals/shield.dm:9]: And here.

**Review by Karmic-Skink — CHANGES_REQUESTED** [review]
Core file changes require tagging.

**Comment by SharkeeByond** [comment]
Is it good now?

File: code/modules/station_goals/bsa.dm [hunk 4]
```diff
@@ -7,10 +7,10 @@
 
 /datum/station_goal/bluespace_cannon/get_report()
 	return {"Our military presence is inadequate in your sector.
-	 We need you to construct BSA-[rand(1,99)] Artillery position aboard your station.
+	 We need you to construct BSA-[rand(1,99)] Artillery position aboard your ship. 
 
 	 Base parts are available for shipping via cargo.
-	 -Nanotrasen Naval Command"}
+	 -Nanotrasen Naval Command"} //NSV13
```
Comment (covertcorvid) [inline code/modules/station_goals/bsa.dm:13]:
```suggestion
	 -Nanotrasen Naval Command"} //NSV13 - changed station to ship
```

File: code/modules/station_goals/dna_vault.dm [hunk 5]
```diff
@@ -33,14 +33,14 @@
...
 
 	 The DNA Vault needs to contain samples of:
 	 [animal_count] unique animal data
 	 [plant_count] unique non-standard plant data
 	 [human_count] unique sapient humanoid DNA data
 
-	 Base vault parts are available for shipping via cargo."}
+	 Base vault parts are available for shipping via cargo."} //NSV13
```
Comment (covertcorvid) [inline code/modules/station_goals/dna_vault.dm:43]:
```suggestion
	 Base vault parts are available for shipping via cargo."} //NSV13 - changed station to ship
```

File: code/modules/station_goals/shield.dm [hunk 6]
```diff
@@ -6,11 +6,11 @@
...
 			 We have a prototype shielding system you must deploy to reduce collision-related accidents.
 
 			 You can order the satellites and control systems at cargo.
-			 "}
+			 "} //NSV13
```
Comment (covertcorvid) [inline code/modules/station_goals/shield.dm:13]:
```suggestion
			 "} //NSV13 - changed station to ship
```

**Review by covertcorvid — APPROVED** [review]
A description of what changed is helpful if it's not on the same line as the change

**Pull request merged**